GliaWidgets/Sources/ViewModel/Chat/ChatViewModel.swift

```diff
@@ -239,7 +240,7 @@ class ChatViewModel: EngagementViewModel {
                         self.action?(.scrollToBottom(animated: true))
                     case .failure:
                         guard let self else { return }
-
+                        self.environment.log.prefixed(Self.self).info("Show Unexpected error Dialog")
```

Comment:
do we need to log error here?

Reply:
https://github.com/salemove/ios-sdk-widgets/pull/852#issuecomment-1854455410